code/datums/jobs.dm

```diff
@@ -1487,6 +1487,31 @@ ABSTRACT_TYPE(/datum/job/civilian)
 		src.access = get_access("Mail Courier")
 		return
 
+/datum/job/special/stowaway
+	name = "Stowaway"
+	limit = 2
+	wages = 0
+	low_priority_job = TRUE
+	slot_back = list(/obj/item/storage/backpack/satchel/anello)
+	slot_belt = list(/obj/item/crowbar)
+	slot_foot = list(/obj/item/clothing/shoes/brown)
+	slot_jump = list(/obj/item/clothing/under/color/grey)
+	slot_head = list(/obj/item/clothing/head/green)
+	slot_ears = list(/obj/item/device/radio/headset/civilian)
+	slot_poc1 = list(/obj/item/currency/spacecash/fivehundred)
+	slot_poc2 = list(/obj/item/scissors)
+	slot_lhan = list(/obj/item/screwdriver)
+
+	items_in_backpack = list(/obj/item/currency/spacecash/buttcoin = 2)
+	rounds_needed_to_play = 10 // It would really suck ass to play this as a new player
```

Comment:
Eh, I'd rather not be adding round reqs to stuff that doesn't *need* it.

Reply:
My concern is that the starting kit expects you to know how to hack doors-- if you don't, you're stuck in a dark, dingy maintenance shaft (most of the time). Unlucky players need to have some level of skill to have a good time with Stowaway, so I do see it as a need...

Reply:
I'm just going to remove this for now, we can re-add it if really needed. We really want to keep round locks to an absolute minimum.